### ISSUE TITLE
Why did you stop publishing in the asset store?

### ISSUE THREAD

**Issue by adarapata** [ssannandeji/Zenject-2019#665]
Hello,
I confirmed that Zenject was suspended from the asset store.

https://assetstore.unity.com/packages/tools/integration/zenject-dependency-injection-ioc-17758

I want you to tell me why.

See also #662

**Comment by extrawurst** [comment]
@adarapata I would not expect an answer here. Just go with Extenject wich is maintained by the original author.

**Comment by Stexe** [comment]
@Extrawurst : Extenject has been removed from the store as well... seems weird. Wonder why this all happened.

EDIT: Looks like Extenject had legal claims filed against them:

> Extenject has been removed from Asset Store because of a copyright claim by Modest Tree, so the only way to obtain it currently is through the releases section on github.
> 
> Update: Extenject is available in the Unity Asset Store
> 
> Modest Tree has also blocked me from contributing to zenject github issues and removed me from the zenject gitter chat, so to get my help please report issues to extenject, and discuss in the extenject chat instead
> 
> They have also filed a lawsuit against me and Unity tells me that they will not re-enable Extenject in the Asset Store until the lawsuit is complete, which might not occur until 2020.
> 
> I have been advised not to comment on details of the lawsuit right now - however you can read their filing and my defense (both of which are public) for details if interested. The lawsuit also relates to other MIT license open source projects I created such as Projeny and Unity3dAsyncAwaitUtil

**Comment by Mathijs-Bakker** [comment]
Unity removed it from the store because of the following reasons:
Quote/
```
Zenject Dependency Injection IOC
Modest Tree Media
Unfortunately, Zenject Dependency Injection IOC is no longer available.
This package has been deprecated from the Asset Store. This means that new purchases of the package are not allowed and that only users who already purchased or downloaded the package before it was deprecated, are allowed to download it.
In most cases, package deprecation happens because the publisher is unable or unwilling to support the package anymore. We suggest looking for alternative packages or contacting the publisher directly.
If you've already purchased it and need to download a copy, you can do so here.
```
/Unquote

The latest release (currently: 9.1.0) is available in UAS.

**Comment by Stexe** [comment]
@Mathijs-Bakker : Yes, but that doesn't really make sense as the package has been updated regularly as far as I can tell. I can only assume it has to do with the legal issues or some dispute with Unity, but I haven't seen verification for either.

**Comment by j4cky11** [comment]
Original author was sued by Modest Tree after leaving the company and forking his own version. Yes, they sued againts a MIT license :D
